scripts/bump_release.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -o pipefail
+set -e
+set -x
+
+NEW_GARMADON_RELEASE=$1
+cd "`dirname "$0"`/.."
+
+# Create/Switch to release branch locally
+git checkout release_$NEW_GARMADON_RELEASE || git checkout -b release_$NEW_GARMADON_RELEASE
```

Comment:
What happens if both fail? Shouldn't we exit the script?

Reply:
set -e -> Exit immediately if a command exits with a non-zero status.

Reply:
True that!